glide.yaml

```diff
@@ -24,7 +24,7 @@ import:
     version: ^2.19.0
   - package: github.com/ghodss/yaml
   - package: github.com/Masterminds/semver
-    version: ~1.3.1
+    version: ~1.4
```

Comment:
Could you set this to the latest: 1.4.2 ?

Reply:
Done 👍